pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/SierraTransformer.scala

```diff
@@ -0,0 +1,33 @@
+package uk.ac.wellcome.platform.transformer.sierra
+
+import grizzled.slf4j.Logging
+
+import uk.ac.wellcome.models.transformable.sierra.SierraBibNumber
+import uk.ac.wellcome.platform.transformer.sierra.source.SierraBibData
+
+trait Info {
+  def info: String
+}
+
+trait SierraTransformer extends Logging {
+
+  type Output
+
+  type TransformInfo <: Info
```

Comment:
Is there a reason to not just have `def info` that returns a `String`?

Reply:
One problem is that we are generating the info inside the `transform` method, so if we had a separate method just for an `info` string, it would essentially have to duplicate the logic inside `transform`.

We could build a string directly inside `transform`, and return `(Output, String)` (instead of `(Output, TransformInfo)`), but having an associated type I think is cleaner and is easier to e.g. export as json if we wanted to represent it differently.